Assets/_Project/Scripts/UserInterface/Views/ApplicationInfoView.cs

```diff
@@ -0,0 +1,42 @@
+using PolSl.UrbanHealthPath.UserInterface.Components;
+using PolSl.UrbanHealthPath.UserInterface.Initializers;
+using PolSl.UrbanHealthPath.UserInterface.Interfaces;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Views
+{
+    public class ApplicationInfoView : MonoBehaviour, IDisplayable, IInitializable
+    {
+        [SerializeField] private Button backButton, forwardButton;
```

Comment:
Shouldn't these declarations be separate?

Reply:
TBH I don't have opinion on such declarations but with attributes I think one per line is more readable. What do you think?